stylesheets/landing.css

```diff
@@ -0,0 +1,210 @@
+/* This stylesheet will contain all the styling for the landing page */
+
+/* Project was designed with a mobile first approach */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  overflow-x: hidden;
+}
+
+h1 {
+  font-family: 'Crete Round', serif;
+}
+
+p {
+  font-family: 'Roboto', sans-serif;
+}
+
+/* actual page styling */
+
+/* Header section */
+
+.header {
+  height: 92px;
+  width: 375px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  background-color: #3c3a39;
+}
+
+.topHeaderBar {
+  margin: 0 10px 10px 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 10px;
+  color: #fff;
+}
+
+.authorheader h2 {
+  font-size: 52px;
+  font-family: 'Crete Round', serif;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6800;
+}
+
+.topnavBar {
+  margin: 10px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #fff;
+}
+
+.topnavBar .navleft h2 {
+  position: static;
+  height: 24px;
+  left: 12px;
+  top: 10px;
+  font-size: 18px;
+  font-weight: 700;
+  font-style: normal;
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+  text-align: center;
+}
+
+/* .topnavBar .navleft h2 a {
+  text-decoration: none;
+  color: #fff;
+} */
+
+/* main page styling */
+.main {
+  height: 590px;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  background: #1c1a19;
+  overflow: hidden;
+
+  /* position: relative; */
+}
+
+.authorheader {
+  margin: 0 24px 12px 24px;
+  padding-top: 120px;
+  position: relative;
+}
+
+.authordetails {
+  border-radius: 0;
+  width: 375px;
+  overflow: hidden;
+}
+
+.bg img {
+  position: absolute;
+  top: 52%;
+  left: 50px;
+  right: 5%;
+  bottom: 0;
+  height: 311.7103797638283px;
+  width: 400.355061045224px;
+  z-index: 1;
+  border-radius: 0;
+  transform: matrix(-0.4, 0.3, 0.4, 0.7, 120, -20);
+}
+
+.authoroccupation {
+  margin: 0 24px 12px 24px;
+  height: 27px;
+  width: 327px;
+  left: 24px;
+  top: 256px;
+}
+
+.authoroccupation p {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  font-style: normal;
+  color: #fff5e1;
+}
+
+.authorexplanation {
+  width: 327px;
+  height: 124px;
+  margin: 0 24px 12px 24px;
+}
+
+.authorexplanation p {
+  font-family: 'Roboto', sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  font-style: normal;
+  color: #fff;
+}
```

Comment:
- [OPTIONAL]

- [ ] I can see that the correct `font-weight` in the Figma template is `400`. please consider adding the correct font weight so that your design matches that of your chosen template.